deployments/polygon/usdc/migrations/1681313581_more_comp.ts

```diff
@@ -0,0 +1,78 @@
+import { Contract } from 'ethers';
+import { DeploymentManager, migration } from '../../../../plugins/deployment_manager';
+import { calldata, exp, proposal } from '../../../../src/deploy';
+
+import { expect } from 'chai';
+
+const ERC20PredicateAddress = '0x40ec5B33f54e0E8A33A975908C5BA1c14e5BbbDf';
+const RootChainManagerAddress = '0xA0c68C638235ee32657e8f720a23ceC1bFc77C77';
+
+export default migration('1661899622_rewards', {
+  async prepare(deploymentManager: DeploymentManager) {
+    return {};
+  },
+
+  async enact(deploymentManager: DeploymentManager, govDeploymentManager: DeploymentManager) {
+    const trace = deploymentManager.tracer();
+    const ethers = deploymentManager.hre.ethers;
+
+    const {
+      governor,
+      comptrollerV2,
+      COMP,
+    } = await govDeploymentManager.getContracts();
+
+    const {
+      rewards,
+    } = await deploymentManager.getContracts();
+
+    const RootChainManager = await deploymentManager.existing(
+      'RootChainManager',
+      RootChainManagerAddress
+    );
+    const COMPAmountToBridge = exp(10_000, 18); // XXX
+    const depositCOMPData = ethers.utils.defaultAbiCoder.encode(['uint256'], [COMPAmountToBridge]);
+    const depositForCOMPCCalldata = ethers.utils.defaultAbiCoder.encode(
+      ['address', 'address', 'bytes'],
+      [rewards.address, COMP.address, depositCOMPData]
+    );
+
+    const actions = [
+      // 4. Approve Polygon's ERC20Predicate to take Timelock's COMP (for bridging)
```

Comment:
Should this be `1.` instead of `4.`?

Reply:
> Seems like the scenario is not recognizing this as a cross-chain proposal. I think you'll have to add the `RootChainManager` address to the check here:
> 
> https://github.com/compound-finance/comet/blob/9ce9aa93222ec05d5414df23427f6bebf60622a1/scenario/utils/isBridgeProposal.ts#L16

Hm are we even storing this address to check?

Reply:
I clicked reply on Kevin's comment, not sure how it ended up here!

Reply:
> Hm are we even storing this address to check?

I don't think it's currently stored, but we should probably add it to roots like we did for the other cross-chain entry point contracts. Though that makes me want to re-think where we should be storing these types of addresses, since roots can become bloated over time by contracts not related directly to Comet.